tests/utils/test_file_utils.py

```diff
@@ -49,6 +49,26 @@ def edit_func(old_dict):
     assert "more_text" not in file_utils.read_yaml(temp_dir, yaml_file)
 
 
+def test_yaml_write_unsorted(tmpdir):
+    temp_dir = str(tmpdir)
+    yaml_file = random_file("yaml")
+    data = {
+        "a": 1,
+        "c": 2,
+        "b": 3,
```

Comment:
Might be a dumb question but should we put the test data out of order to clarify the test?

Reply:
The ordering is determined by keys, so the data dictionary is actually out of order. I've updated this test to test both the `sort_keys=True` and `sort_keys=False` cases.

Reply:
Oh sorry I misread the keys as a b c 🤦 Sorry! But the more tests the better :)